CI/translator.py

```diff
@@ -0,0 +1,141 @@
+"""Translate missing msgstr entries in .po files using the specified translator."""
+
+import logging
+import re
+import time
+from glob import glob
+from pathlib import Path
+
+import deep_translator
+
+logger = logging.getLogger(__name__)
+
+
+def translate_missing_po_entries(  # noqa: C901
```

Comment:
Can you add a field for passing an API access key here? I imagine that would simply be passed along at line 47.

Reply:
This would be in the kwargs, yes.